src/examples/server/sseAndStreamableHttpCompatibleServer.ts

```diff
@@ -0,0 +1,316 @@
+import express, { Request, Response } from 'express';
+import { randomUUID } from "node:crypto";
+import { McpServer } from '../../server/mcp.js';
+import { EventStore, StreamableHTTPServerTransport } from '../../server/streamableHttp.js';
+import { SSEServerTransport } from '../../server/sse.js';
+import { z } from 'zod';
+import { CallToolResult, isInitializeRequest, JSONRPCMessage } from '../../types.js';
+
+/**
+ * This example server demonstrates backwards compatibility with both:
+ * 1. The deprecated HTTP+SSE transport (protocol version 2024-11-05)
+ * 2. The Streamable HTTP transport (protocol version 2025-03-26)
+ * 
+ * It maintains a single MCP server instance but exposes two transport options:
+ * - /mcp: The new Streamable HTTP endpoint (supports GET/POST/DELETE)
+ * - /sse: The deprecated SSE endpoint for older clients (GET to establish stream)
+ * - /request: The deprecated POST endpoint for older clients (POST to send messages)
```

Comment:
Nit: we've usually named this `/message`, so we might want to match that for clarity (even though it doesn't matter semantically):
```suggestion
 * - /message: The deprecated POST endpoint for older clients (POST to send messages)
```